com.unity.shadergraph/Editor/GraphUI/GraphElements/Stylesheets/ShaderGraphMainToolbar_dark.uss

```diff
@@ -0,0 +1,15 @@
+.ge-main-toolbar #optionsButton .icon {
```

Comment:
Should the style selectors in `ShaderGraphMainToolbar_dark.uss` and `ShaderGraphMainToolbar_light.uss` be the same?

Reply:
A good question, I think it should be fine cause we actually change which stylesheet is loaded in based on the users current skin (light mode/dark mode) etc in the `ShaderGraphMainToolbar`

`            this.AddStylesheet(EditorGUIUtility.isProSkin ? "ShaderGraphMainToolbar_dark.uss" : "ShaderGraphMainToolbar_light.uss");`

So based on whatever their active mode is, the appropriate stylesheet gets loaded in and used, so there won't be any conflicts

Though say they change their active mode in the editor, thats another question, hopefully that just triggers a whole UI rebuild and we're good but I didn't think that far ahead with this

Reply:
also we don't really have the appropriate icons for the light/dark modes at the moment, so it'd look wrong in any case

Reply:
I was assuming that these two files would define the same selector paths but they are currently different. Does this mean that different buttons show up in light and dark mode?